addons/mail/static/src/core/common/autoresize_input.js

```diff
@@ -49,28 +43,26 @@ export class AutoresizeInput extends Component {
         });
     }
 
+    onBlur() {
+        if (this.focused) {
+            this.props.onValidate(this.state.value);
+            this.state.value = this.props.value;
```

Comment:
This line (already here before diff) makes no sense.
Resetting the input when actually saving??